File: hw/ramos/u2/hw25MongoDBConnection/python/controller/ContactController.py
import sys
import os
sys.path.append(os.path.dirname(os.path.dirname(os.path.abspath(__file__)))) 
sys.path.append(os.path.join(os.path.dirname(os.path.dirname(os.path.abspath(__file__))), "model"))
from model.Contact import Contact
from controller.ContactDAO import ContactDAO

class ContactController:

    def __init__(self):
        self.dao = ContactDAO()

    def add_contact(self, id, first_name, last_name, birth_date, age, type_of_contact, sex, hobbies, comments):
        contact = Contact(
            id=id,
            first_name=first_name,
            last_name=last_name,
            birth_date=birth_date,
            age=age,
            type_of_contact=type_of_contact,
            sex=sex,
            hobbies=hobbies,
            comments=comments
        )

        return self.dao.save(contact)

File: hw/rojas/u2/hw24ProjectGUIs/ContactstBook/src/main/java/ec/edu/espe/contactstbook/controller/ContactController.java
package ec.edu.espe.contactstbook.controller;

import ec.edu.espe.contactstbook.model.Contact;
import java.util.ArrayList;
import java.util.Date;

/**
 *
 * @author Josue Rojas
 */
public class ContactController {

    private ContactDAO dao = new ContactDAO();
    
    public boolean addContact(int id, String firstName, String lastName, Date birthDate, int age, String typeOfContact, String sex, ArrayList<String> hobbies, String comments){
        Contact contact = new Contact(id, firstName, lastName, birthDate, age, typeOfContact, sex, hobbies, comments);
        return dao.save(contact);
    }
    
}
